tests/behat/features/cci/cci-homepage.feature

```diff
@@ -56,7 +56,7 @@ Feature: verify specific links on homepage
     # Then I should be on "https://ask.cyberinfrastructure.org/c/communities/rmacc/65"
 
     When I am on the homepage
-    And I wait for the page to be loaded
+    And I wait for 2 seconds
```

Comment:
This was due to a VM at WPI that we use to do redirects that was down. Should be fine to say wait for the page to be loaded.